Sources/View/UITableView+Reusable.swift

```diff
@@ -44,28 +52,37 @@ public extension UITableView {
 
    - note: The `cellType` parameter can generally be omitted and infered by the return type,
    except when your type is in a variable and cannot be determined at compile time.
-   - seealso: `dequeueReusableCellWithIdentifier(_:,forIndexPath:)`
+   - seealso: `dequeueReusableCell(withIdentifier:,for:)`
    */
   final func dequeueReusableCell<T: UITableViewCell where T: Reusable>(indexPath: IndexPath, cellType: T.Type = T.self) -> T {
```

Comment:
`UITableView` has method `-dequeueReusableCell(withIdentifier: String, for: IndexPath)`, so I would suggest `dequeueReusableCell(for: IndexPath)` instead `dequeueReusableCell(indexPath: IndexPath)`